index.html

```diff
@@ -26,7 +26,6 @@
         id: "mywindow",
```

Comment:
Index.html still contains hardcoded string ID of the window
- you can use the same variable here
- _or_ you can move your `webix.ui` calls from .html to your .js file